contributing/commit-messages.md

```diff
@@ -0,0 +1,122 @@
+[&larr; Table of Contents](../CONTRIBUTING.md)
+
+# Commits
+
+FDS uses conventional commits to produce release versions according to semantic release.
+
+Your commit messages must have a shape like
+
+```
+<type>[optional scope]: <description>
```

Comment:
should the optional scope have parenthesis around it instead of brackets?

Reply:
We should also maybe start with an example, and then flesh out the structure of it. In general, we should always start simple and real world, and then break it down technically.